CHANGELOG.rst

```diff
@@ -4,5 +4,6 @@ Changelog
 master
 ------
 
+(`#52 <https://github.com/OMS-NetZero/FAIR/pull/52>`) Fixed bug that didn't pick up first year of CO2 concentrations and radiative forcing after restart_in
```

Comment:
is it worth mentioning here that you've now changed internal CO2 concentrations to be absolute, rather than anomalies?

Reply:
Yep - now updated